math/minuit2/src/Minuit2Minimizer.cxx

```diff
@@ -1201,7 +1204,43 @@ bool Minuit2Minimizer::Hesse()
       return false;
    }
 
-   const int strategy = Strategy();
+
+   // set strategy and add extra options if needed
```

Comment:
It would be good if the code is not duplicated from `Minimize()`, but I can also take care of this technicality.